### ISSUE TITLE
Please add new url www.trainedbyjp.com

### ISSUE THREAD

**Issue by oxfordgrad** [rg3/youtube-dl#26403]
## Checklist

- [x ] I'm reporting a new site support request
- [x ] I've verified that I'm running youtube-dl version **2020.07.28**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that none of provided URLs violate any copyrights
- [ x I've searched the bugtracker for similar site support requests including closed ones

## Example URLs

- Single video: https://www.trainedbyjp.com/legs-with-dorian-yates/
- Single video: https://www.trainedbyjp.com/eat-big-to-get-big/
- Playlist: NO PLAYLIST

## Description

WRITE DESCRIPTION HERE

Please can you add https://www.trainedbyjp.com

**Comment by blackjack4494** [comment]
seems premium account is needed.

**Comment by oxfordgrad** [comment]
> seems premium account is needed.

Yes I am a member, i have login details etc, can this site added so i can download some content?
